annotation/src/main/java/org/cbioportal/genome_nexus/annotation/GenomeNexusAnnotation.java

```diff
@@ -55,8 +61,14 @@
  */
 @SpringBootApplication // shorthand for @Configuration, @EnableAutoConfiguration, @ComponentScan
 @EnableSwagger2 // enable swagger2 documentation
-public class GenomeNexusAnnotation extends SpringBootServletInitializer
+public class GenomeNexusAnnotation extends SpringBootServletInitializer implements CommandLineRunner
```

Comment:
Would it be too difficult to implement the `CommandLineRunner` in a separate class? Something like `class GenomeNexusImporter implements CommandLineRunner`.